foreshadow/cleaners/internals/datetimes.py

```diff
@@ -0,0 +1,46 @@
+"""Transforms for datetime inputs."""
+import re
+
+from foreshadow.cleaners.data_cleaner import BaseCleaner
+
+
+def _split_to_new_cols(t, return_search=False):
+    """Clean text if it is in a YYYYMDD format and split to three columns.
```

Comment:
I know this is a proof of concept but can we use the datetime library that will parse any date and will allow us to extract the entities we need?